mava/utils/jax_training_utils.py

```diff
@@ -44,34 +44,53 @@ def construct_norm_axes_list(
     start_axes: int, axes_list: List[Any], obs_shape: Tuple
 ) -> Tuple[slice, ...]:
     """Construt a list of Tuples containing the features on which to apply normalisation
-
-    start_axes (int) --- default axes from which to start
-    axes_list (List[int]) --- user specified
-    obs_shape (Tuple) --- observations shape
+    Args:
+        start_axes (int) --- default axes from which to start
+        axes_list (List[int]) --- user specified
+        obs_shape (Tuple) --- observations shape
 
     Returns:
         axes_list: a tuple to be used with np.r_
+
+    The start_axes is 0 unless we use a wrapper like concat_agent_id
+    or concat_previous_actions which add one hot encorded vectors
+    at the start of the array.
+    axes_list corresponds to user specified axes we want to normalise
+    We aussume the user does not consider contenation when specifying axes_list.
+    If axes_list is empty then we nornmalise all the axes
```

Comment:
I think a slight function change here would be useful, if axes_list is `None` then normalize everything and if it is empty normalize nothing.

Reply:
The code has been modified accordingly to follow this behaviour. Nice catch actually. Thanks @sash-a